README.md

```diff
@@ -138,6 +138,7 @@ Check the prerequisite:
 
 * [Rust toolchain](https://www.rust-lang.org/tools/install) (nightly)
 * [Protobuf compiler](https://grpc.io/docs/protoc-installation/) (>= 3.15)
+* C/C++ building essentials, including `gcc`/`g++`/`autoconf` and glibc library (eg. `libc6-dev` on Ubuntu and `glibc-devel` on Fedora)
```

Comment:
Do we need to use the `build-essential` package as an example?
https://docs.greptime.com/contributor-guide/getting-started#build-dependencies

Reply:
> Do we need to use the `build-essential` package as an example? https://docs.greptime.com/contributor-guide/getting-started#build-dependencies

`build-essential` is only available on Debian based distros like Ubuntu.